src/OmniSharp.Abstractions/Models/v1/FilesChanged/FilesChangedRequest.cs

```diff
@@ -1,8 +1,11 @@
-using System.Collections.Generic;
 using OmniSharp.Mef;
+using System.Collections.Generic;
```

Comment:
Sort `System` namespaces first please. You've got the option set incorrectly in VS (this is also the preference in Roslyn).